docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md

```diff
@@ -135,6 +136,50 @@ Some widely used Outlook add-in publishers have already updated their add-ins as
 
 If the publisher updated their manifest, and the add-in is deployed through the Microsoft store, you'll be prompted as an administrator to upgrade and deploy the updates. If the publisher updated their manifest, and the add-in is deployed through central deployment, you'll need to deploy the new manifest as an administrator. In some cases the publisher may have an admin consent URI you need to use to consent to new scopes for the add-in. Reach out to publishers if you need more information about updating an add-in.
 
+### Some add-ins are breaking. Can I tell if this is because Exchange tokens were turned off?
+
+Beginning February 17 2025, Microsoft is rolling out a change to gradually turn off legacy Exchange Online tokens for all users. The change won't turn off Exchange tokens in your tenant if you already [turned legacy Exchange Online tokens on](turn-exchange-tokens-on-off.md).
+
+If your tenant uses an add-in that still relies on Exchange tokens, the add-in will break or lose functionality. Because the change is rolled out per user, it's possible for one, or a few users to see an add-in affected when Exchange tokens are off. But the add-in can continue working for other users who have Exchange tokens on. If you notice that an add-in has issues, and suspect it may be affected by Exchange tokens turned off, please take the following actions.
```

Comment:
Some suggestions to make it a little clearer that admins could see mixed behavior.
```suggestion
If your tenant uses an add-in that still relies on Exchange tokens, the add-in will break or lose functionality. The update is rolled out **per user**. This means that one or more users may have an add-in affected when Exchange tokens are off, but other users would still have a working add-in. If you notice that an add-in has issues and suspect it may be affected by Exchange tokens turned off, please take the following actions.
```